front/src/modules/ui/menu-item/components/__stories__/MenuItemCommand.stories.tsx

```diff
@@ -24,6 +25,12 @@ export const Default: Story = {
     text: 'First option',
     command: '⌘ 1',
   },
+  render: (props) => (
+    <Command>
+      {/* eslint-disable-next-line twenty/no-spread-props */}
```

Comment:
let's avoid doing it even in the tests. I think you can unconstruct props instead!